core/variant/array.cpp

```diff
@@ -202,9 +202,10 @@ Error Array::resize(int p_new_size) {
 	return _p->array.resize(p_new_size);
 }
 
-void Array::insert(int p_pos, const Variant &p_value) {
+Error Array::insert(int p_pos, const Variant &p_value) {
 	ERR_FAIL_COND(!_p->typed.validate(p_value, "insert"));
```

Comment:
`ERR_FAIL_COND`  doesn't return value and should be changed to `ERR_FAIL_COND_V`